### PR TITLE
feat: add organization page into identity page

### PR DESCRIPTION
**Pull request by yusualhashash**

### CODE REVIEW

File: apps/web/src/app/[lang]/(main)/dashboard/[domain]/organization/action.ts
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/(main)/dashboard/[domain]/organization/action.ts:1]: Can we use one fetch function in this file, with generic link? and have the other functions as wrappers if you want, or even use it immediately in the project?

File: apps/web/src/app/api/organization/[data]/route.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+type Clients = {
+  [key: string]: any;
+};
+
+const errorResponse = (message: string, status: number = 400) =>
+  new Response(JSON.stringify({ message }), { status: status });
+
+function isApiError(error: unknown): error is ApiError {
+  return error instanceof ApiError;
+}
```
Comment (a0m0rajab) [inline apps/web/src/app/api/organization/[data]/route.ts:15]: Can we use this as imports from the `api/admin/[data]/route.ts` file?

File: apps/web/src/app/[lang]/(main)/dashboard/[domain]/organization/page.tsx [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+          action={MoveAllUserAction}
+        />
+      )}
+      {isSubUnitOpen && (
+        <AutoformDialog
+          open={isSubUnitOpen}
+          onOpenChange={setIsSubUnitOpen}
+          action={AddSubUnitAction}
+        />
+      )}
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/(main)/dashboard/[domain]/organization/page.tsx:728]: Can we use one autoform dialog?

**Review by a0m0rajab — CHANGES_REQUESTED** [review]

File: apps/web/src/app/[lang]/(main)/dashboard/[domain]/organization/action.ts
Comment (yusualhashash) [reply apps/web/src/app/[lang]/(main)/dashboard/[domain]/organization/action.ts:1]: done
image

File: apps/web/src/app/api/organization/[data]/route.ts (hunk 1 above)
Comment (a0m0rajab) [reply apps/web/src/app/api/organization/[data]/route.ts:15]: You can export the function or move it to `utils.ts`.

**Review by a0m0rajab — APPROVED** [review]

**Pull request merged**